github/codeowners.py

```diff
@@ -155,6 +156,19 @@ def determine_email_address(
     return EmailAddress(user.email)
 
 
+def usernames_with_email_address(
```

Comment:
this function determines gh-usernames from email-addresses - this should be reflected in the name

examples that come to my mind:
- iter_usernames
- find_github_usernames
- [github_]usernames_from_email_address
- to_github_usernames